### PR TITLE
werewolf-4 Add Day View

### PR DESCRIPTION
**Pull request by nickydonna**
Related to #4

### CODE REVIEW

File: controlpad/controller/stores/game.js [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
 export const showCreditsAction = () => Action.create(ActionNames.P_SHOW_CREDITS)
 
+/** @enum {string} */
+export const Role = {
+    KILLER: 'KILLER',
+    VILLAGER: 'VILLAGER',
+};
+
+/** @enum {string} */
+export const GameState = {
```
Comment (safwaan-menk) [inline controlpad/controller/stores/game.js:17]: Looks good

File: controlpad/controller/stores/game.js [hunk 2]
```diff
@@ -18,23 +35,20 @@ export const showCreditsAction = () => Action.create(ActionNames.P_SHOW_CREDITS)
...
-export function dispatch(action) {
-    const {type, payload} = action;
-    switch (type) {
-        case ActionNames.P_CONNECTED:
-            $game.setKey('connected', true);
-            break;
-        case ActionNames.P_SHOW_RULES:
-        case ActionNames.P_SHOW_CREDITS:
-        case ActionNames.P_START_GAME:
-            channel.sendMessage(action.toString());
```
Comment (safwaan-menk) [inline controlpad/controller/stores/game.js:35]: Should be using send_controlpad_message here

**Comment by safwaan-menk** [comment]
Changes look good to me!

**Review by safwaan-menk — APPROVED** [review]

File: controlpad/controller/stores/game.js (hunk 2 above)
Comment (nickydonna) [reply controlpad/controller/stores/game.js:35]: I have reowrked this in the next PE

**Pull request merged**